x-pack/heartbeat/Jenkinsfile.yml

```diff
@@ -23,6 +23,7 @@ stages:
         stage: checks
     unitTest:
         mage: "mage build unitTest"
+        withNodejs: true
         stage: mandatory
     goIntegTest:
         mage: "mage goIntegTest"
```

Comment:
Can we also add this to `goIntegTest` ? https://github.com/elastic/beats/pull/30632 which is due to be merged relatively soon adds integration tests that would also benefit here

Reply:
Yes, see https://github.com/elastic/beats/pull/32710/commits/01aae7bb154e16008150abcce112c7ac87ec5474